doc/source/whatsnew/v0.25.1.rst

```diff
@@ -102,9 +102,10 @@ MultiIndex
 
 I/O
 ^^^
+
 - Avoid calling ``S3File.s3`` when reading parquet, as this was removed in s3fs version 0.3.0 (:issue:`27756`)
 - Better error message when a negative header is passed in :func:`pandas.read_csv` (:issue:`27779`)
--
+- Fix bug in :meth:`io.json.json_normalize` when nested meta paths with a nested record path. (:issue:`27220`)
```

Comment:
Move to 0.25.2 at this point

Reply:
Just FYI, I think we're being conservative with 0.25.2 backports. Just 3.8 compat and fixes for regressions in 0.25.

Reply:
Sounds good - @yanglinlee let's go 1.0.0 instead then